docs/resources/Auto_Moderation.md

```diff
@@ -168,11 +170,12 @@ An action which will execute whenever a rule is triggered.
 
 ###### Action Types
 
-| Action Type        | Value | Description                                                                                                                                                |
-| ------------------ | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| BLOCK_MESSAGE      | 1     | blocks a member's message and prevents it from being posted. A custom explanation can be specified and shown to members whenever their message is blocked. |
-| SEND_ALERT_MESSAGE | 2     | logs user content to a specified channel                                                                                                                   |
-| TIMEOUT            | 3     | timeout user for a specified duration *                                                                                                                    |
+| Action Type              | Value | Description                                                                                                                                                |
+| ------------------------ | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| BLOCK_MESSAGE            | 1     | blocks a member's message and prevents it from being posted. A custom explanation can be specified and shown to members whenever their message is blocked. |
+| SEND_ALERT_MESSAGE       | 2     | logs user content to a specified channel                                                                                                                   |
+| TIMEOUT                  | 3     | timeout user for a specified duration *                                                                                                                    |
+| BLOCK_MEMBER_INTERACTION | 4     | prevents a member from using text, voice, or other interactions                                                                                            |
```

Comment:
Is `BLOCK_MEMBER_INTERACTION` just a timeout without a duration? I'm not sure I understand how it is different to timeouts.